<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reglages_courriers</class>
 <widget class="QWidget" name="reglages_courriers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Elephant</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Réglages Courriers</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>451</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Définir nombre de courriers avant archivage  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxnbrecourrier">
         <property name="font">
          <font>
           <family>Times New Roman</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Archiver les courriers chaque</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxsequencedarchivage">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Mois</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Trimestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Semestre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Année</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 ans</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 ans</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Définir marge d'heures jounalière d'enrégistrement </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEditmarges_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>à</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEditmarges">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="bntenregreglagecour">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enrégistrer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntannrecour">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
